dynamiq/nodes/converters/csv.py

```diff
@@ -30,11 +28,6 @@ class CSVConverterInputSchema(BaseModel):
     files: list[BytesIO | bytes] | None = Field(
         default=None, description="List of file objects or bytes representing CSV files."
     )
-    delimiter: str = Field(default=",", description="Delimiter used in the CSV files.")
```

Comment:
Why can't we now specify such parameters during single execution? Like user can pass different files, but delimiter, content_column should always be predefined during Node configuration?

It seems to make sense to allow to override this, but with default None and after use like:
`delimiter = input_data.delimiter or self.delimiter`